apps/slackbot/Dockerfile

```
@@ -12,13 +12,21 @@ ENV PYTHONUNBUFFERED=1 \
+# Create non-root user for security (must exist before uv sync so --chown works)
+RUN groupadd --gid 1000 slackbot && \
+    useradd --uid 1000 --gid slackbot --shell /bin/bash --create-home slackbot
+
 COPY pyproject.toml uv.lock ./
 COPY packages/db-python ./packages/db-python
 COPY apps/slackbot ./apps/slackbot
 
+# Run uv sync as slackbot so the venv is created with correct ownership;
+# no post-install chown -R needed, avoiding metadata rewrites on the venv tree.
 RUN --mount=type=cache,target=/root/.cache/uv \
+    --chown=slackbot:slackbot \
```

Hadolint (2.15.1)

[error] unexpected "--chown=sla"
expecting "--mount=", "--network=", "--security=", '\', or at least one space

(DL1000)